.eslintrc

```diff
@@ -26,6 +26,7 @@
     "consistent-return": 0,
     "no-underscore-dangle": 0,
     "indent": [2, 2, {"SwitchCase": 1}],
-    "space-before-blocks": [2, "always"]
+    "space-before-blocks": [2, "always"],
+    "no-var": [1]
```

Comment:
@benjefferies There's no reason to use `var` unless you specifically need to take advantage of the way it's scoped—which is highly unlikely.